### ISSUE TITLE
请问我在调整好多次bert参数后，得出的结果都是一样的

### ISSUE THREAD

**Issue by feiyangdezhu** [jiangxinyang227/textClassifier#7]
![)8E${~SHP5MZUVL8Y$W6 @D](https://user-images.githubusercontent.com/36184086/54580897-2b3b0d80-4a45-11e9-90a6-3f0e642ce00f.png)

**Comment by jiangxinyang227** [comment]
你好，你这里的结果很奇怪，从precision和recall来看，正例完全没有被预测出来，也就是真正例TP=0。也就是在预测时，将所有的类别都预测为负例了。但从你的accuracy来看，数据集正例和负例应该差不多数量，所以最大的原因可能时代码的问题。

**Comment by pmm-511** [comment]
请问您这个问题解决了吗？我也遇到类似的问题，望答疑

**Comment by SmartMapple** [comment]
请问解决了吗，我现在也是这种情况。
INFO:tensorflow:***** Eval results *****
INFO:tensorflow:  **eval_accuracy = 0.6764706**
INFO:tensorflow:  eval_auc = 0.5675585
INFO:tensorflow:  eval_precision = 0.71940297
INFO:tensorflow:  **eval_recall = 0.8637993**
INFO:tensorflow:  global_step = 2751
INFO:tensorflow:  loss = 0.62870294
